scenes/game/local_game.gd

```diff
@@ -3216,7 +3318,17 @@ func do_remaining_effects(performing_player : Player, next_state):
 				change_game_state(Enums.GameState.GameState_PlayerDecision)
 				decision_info.type = Enums.DecisionType.DecisionType_ChooseSimultaneousEffect
 				decision_info.player = performing_player.my_id
-				decision_info.choice = effects_to_choose
+				decision_info.choice = []
+				for effect in effects_to_choose:
+					if 'is_negative_effect' in effect and effect['is_negative_effect']:
```

Comment:
This was duplicated somewhere else, good candidate for a function?
It might not be exactly the same, but at least consider if it makes sense.